app/javascript/packs/containers/OptionsContainer.jsx

```diff
@@ -117,7 +117,7 @@ class OptionsContainer extends React.Component {
 function mapStateToProps(state, ownProps) {
   return {
     answer: getAnswerForQuestionAndBuilding(ownProps.question_id, ownProps.building_id, state),
-    dependentQuestions: getDependentQuestionsForOptions(ownProps.options, ownProps.question_type, state)
+    dependentQuestions: getDependentQuestionsForOptionIds(Object.keys(ownProps.options), ownProps.question_type, state)
```

Comment:
Was this file a part of a merge with Helen's branch?

Reply:
Yeah 😟I tried picking out my changes but I realized some of the files have changes from both...can we just merge Helen's branch with master before mine so all the diffs are gone?